ONSdigital/eq-author-app
# Dependabot upgrades 34

### What is the context of this PR?

Lots of dependabot upgrades. 

This PR also includes a configuration file which tells Dependabot to limit the number of open PRs to 10 (total; 3 for the front end, api and publisher and 1 for the root).

### How to review 

Spin up the app and make sure everything works as expected.

You can use this to validate the Dependabot yml: https://dependabot.com/docs/config-file/validator/

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
